### DISCUSSION TITLE
Skeleton Loading Color

### DISCUSSION THREAD

**Discussion by MathiasVoie** [saadeghi/daisyui#2785] [Ideas / feature requests]
You should be able to change the color of the skeleton loading effect. such as **skeleton-[#{DESIRED_COLOR}]**. this would make it more appealing and wont stick out of the webpage's color scheme.

I saw one discussion being opened in nov 2023 but thought id give it more attention.

**Comment by saadeghi** [comment]
I assume skeleton would only make sense to be used on the `base` color background and the current colors are based on that.  
Is there a need for it to be used elsewhere?
